packages/slate-theme/test/currency.spec.js

```diff
@@ -29,13 +29,23 @@ describe('currency.formatMoney', () => {
     expect(value).toBe('$100');
   });
 
-  test('Formats a number 1000001 to a string of "$10,000"', () => {
+  test('Formats a number 1000001 to a string of "$10.000"', () => {
     var value = formatMoney(1000001, '${{amount_no_decimals_with_comma_separator}}');
-    expect(value).toBe('$10,000');
+    expect(value).toBe('$10.000');
   });
 
   test('Formats a number 1000001 to a string of "$10 000"', () => {
     var value = formatMoney(1000001, '${{amount_no_decimals_with_space_separator}}');
     expect(value).toBe('$10 000');
   });
+
+  test('Formats a number of 1000001 to a string of "$10.000,01"', () => {
+    var value = formatMoney(1000001, '${{amount_with_comma_separator}}');
```

Comment:
Same here. By separator, we're referring to the thousand seperator, where in the number `1,000,000` the `,` is the separator.

Reply:
By using `amount_with_comma_separator` the comma is used  to separate decimals 

![screen shot 2017-10-30 at 3 55 03 pm](https://user-images.githubusercontent.com/5134470/32197661-b98bd3b0-bd8a-11e7-8e25-cefcdef87c51.png)

https://help.shopify.com/manual/payments/currency-formatting

{{ amount_with_comma_separator }} |   | 1.134,65
-- | -- | --

![2017-10-30 15 53 14](https://user-images.githubusercontent.com/5134470/32197609-8582a0a8-bd8a-11e7-86e2-51bb74500af9.gif)